src/Modules/OrchardCore.Commerce.Payment.Stripe/Assets/Scripts/stripe-payment-form.js

```diff
@@ -6,21 +6,30 @@ window.stripePaymentForm = function stripePaymentForm(
     urlPrefix,
     errorText,
     missingText,
-    updatePaymentIntentUrl) {
-    const allErrorContainers = [document.querySelector('.message-error')];
-    const form = document.querySelector('.payment-form');
-    const submitButton = form.querySelector('.pay-button-stripe');
-    const payText = submitButton.querySelector('.pay-text');
-    const paymentProcessingContainer = submitButton.querySelector('.payment-processing-container');
-    const stripeElements = stripe.elements({
-        clientSecret,
-    });
-    const payment = stripeElements.create('payment', {
-        fields: {
-            billingDetails: 'never',
-        },
-    });
-    const placeOfPayment = document.querySelector('#payment-form_payment');
+    updatePaymentIntentUrl,
+    validateUrl,
+    paramsUrl,
+    priceUrl,
+    errorContainerSelector,
+    stripeFieldErrorSelector,
+    paymentFormSelector,
+    payButtonSelector,
+    payTextSelector,
+    paymentProcessingContainerSelector,
+    placeOfPaymentSelector,
+    payButtonValueSelector,
+    addressesSelector,
+    addressSelector,
+    addressTitleSelector
```

Comment:
Please add the previously hard coded values as [default values](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters) for these parameters. This way it won't break for people who have a custom `CheckoutStripe` shape override.